src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands.ContainerRegistry.csproj

```diff
@@ -0,0 +1,179 @@
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Bcl.Async.1.0.168\lib\net40\Microsoft.Threading.Tasks.Extensions.Desktop.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Net" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Http.Extensions, Version=2.2.29.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Net.Http.2.2.29\lib\net45\System.Net.Http.Extensions.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System.Net.Http.Primitives, Version=4.2.29.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Net.Http.2.2.29\lib\net45\System.Net.Http.Primitives.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System.Net.Http.WebRequest" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Security" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Commands\GetAzureContainerRegistry.cs" />
+    <Compile Include="Commands\TestAzureContainerRegistryNameAvailability.cs" />
+    <Compile Include="Commands\GetAzureContainerRegistryCredential.cs" />
+    <Compile Include="Commands\UpdateAzureContainerRegistry.cs" />
+    <Compile Include="Commands\RemoveAzureContainerRegistry.cs" />
+    <Compile Include="Commands\NewAzureContainerRegistry.cs" />
+    <Compile Include="Commands\UpdateAzureContainerRegistryCredential.cs" />
+    <Compile Include="Models\DeploymentTemplate.cs" />
+    <Compile Include="Models\PSContainerRegistry.cs" />
+    <Compile Include="Models\ContainerRegistryClient.cs" />
+    <Compile Include="Models\ContainerRegistryCmdletBase.cs" />
+    <Compile Include="Models\PSContainerRegistryCredential.cs" />
+    <Compile Include="Models\ResourceManagerClient.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Microsoft.Azure.Commands.ContainerRegistry.dll-Help.xml">
```

Comment:
@djyou we no longer have dll-Help.xml files in the repository, so you can remove this after pulling the latest changes from the `preview` branch